backend/src/main/java/com/example/demo/todo/TodoService.java

```diff
@@ -27,11 +27,15 @@ public Optional<Todo> findById(String id) {
 
     public Optional<Todo> updateTodo(String id, String email, Todo updatedTodo) {
                 Optional<Todo> todoFromDB = todoRepository.findByIdAndUserEmail(id, email);
-                // abfrafe zum verglichen kommt hier
-                        return
+                int currentScore = 0;
+                if(updatedTodo.getStatus() != todoFromDB.get().getStatus()){
```

Comment:
hier musst du auf passen: wenn todoFromDB ein leeres Optional ist, dann würde .get() ein NullPointer Exception werfen. Du könntest vor diesem if nochmal mit `if(todoFromDB.isPresent){...}` und dann das nächste if darein setzen

Reply:
alles klar !

Reply:
das geht in der stelle nicht